scripts/fix_unclaimed_records.py

```diff
@@ -0,0 +1,35 @@
+import sys
```

Comment:
If this should be run for everyone, would it make sense to make this a migration? Or if this a one time script or will this be run periodically? If it's a one time guy should move under `/remove_after_use/`?